core/common/src/main/java/alluxio/metrics/Metric.java

```diff
@@ -209,12 +209,26 @@ private String constructFullMetricName() {
   }
 
   /**
-   * Gets the metric name with the appendix of tags. The returned name is of the pattern
-   * name[.tagName:tagValue]*.
+   * Gets the metric name with the appendix of tags.
+   * <p>
+   * Returns the metric name with the provided tags.
+   * The returned name is of the pattern is
+   * {@code name[.tagName:tagValue]*}.
```

Comment:
are you sure about the format ?

Reply:
The pattern was taken from the original documentation.